demo/App.js

```diff
@@ -1,12 +1,177 @@
 import React from "react";
 import { StyleSheet, Text, View } from "react-native";
 import { AppLoading, Asset, Font } from "expo";
+import "ethers/dist/shims.js";
+import { ethers } from "ethers";
+
 
 export default class App extends React.Component {
   state = {
     isLoadingComplete: false
   };
 
+
+	componentDidMount() {
+
+		let ropstenProvider = ethers.getDefaultProvider('ropsten');
```

Comment:
@marcelomorgado I think this would be better as a `const` instead of a `let`, since it doesn't change.

Reply:
Ok